### PR TITLE
feat (cli): shell command

### PR DESCRIPTION
**Pull request by yesnault**

### CODE REVIEW

**Comment by yesnault** [comment]
preview: https://asciinema.org/a/jjiEYt3bGSG3GcCDbIv2Vk3Bm

**Comment by yesnault** [comment]
wip, autocomplete is coming

**Comment by yesnault** [comment]
with autocomplete: https://asciinema.org/a/x1arL3iiNXLW8iMMXgwCmBoxa

File: cli/cdsctl/shell.go [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+			current.environment = args[0]
+		case shellInPipelines:
+			current.position = shellInPipeline
+			current.pipeline = args[0]
+		}
+		return nil
+	}
+
+	lsCommand = func(args []string, current *shellCurrent) *cobra.Command {
+		if current.position == shellInProjects {
```
Comment (bnjjj) [inline cli/cdsctl/shell.go]: switch case

File: cli/cdsctl/shell.go [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+ d8"'    ` + "`" + `"8b  88      ` + "`" + `"8b   d8"     "8b  
+d8'            88        ` + "`" + `8b  Y8,          
+88             88         88  ` + "`" + `Y8aaaaa,    
+88             88         88    ` + "`" + `"""""8b,  
+Y8,            88         8P          ` + "`" + `8b  
+ Y8a.    .a8P  88      .a8P   Y8a     a8P  
+  ` + "`" + `"Y8888Y"'   88888888Y"'     "Y88888P"   
+						
+  
+connecting to cds api...
```
Comment (bnjjj) [inline cli/cdsctl/shell.go]: could be useful to indicate between () the api url

**Comment by yesnault** [comment]
new record: https://asciinema.org/a/fTFpJ5uqClJ0Oq2EsiejGSeBk

**Pull request merged**